bootstrap.py

```diff
@@ -218,6 +218,45 @@ def install_prereqs():
         delete_file('/etc/yum.repos.d/katello-client-bootstrap-deps.repo')
 
 
+def get_puppet_path():
```

Comment:
this would be then (if we use the RPMdb)

```python
puppet_major_version = get_puppet_version()
if puppet_major_version == 3:
    return '/usr/bin/puppet'
elif puppet_major_version in [4,5]:
    return '/opt/puppetlabs/puppet/bin/puppet'
else:
    error